package.json

```diff
@@ -40,9 +39,7 @@
     "jest": "^29.5.0",
     "jest-environment-jsdom": "^29.5.0",
     "lerna": "^6.4.0",
-    "react": "^18.2.0",
-    "react-native": "^0.72.3",
-    "rollup": "^3.18.0",
+    "rollup": "^3.23.0",
```

Comment:
We need at least `v3.23.0` for the `prebuilt-chunk` type used here:

https://github.com/bugsnag/bugsnag-js-performance/blob/d1a7bff635829b206a9c57e79a6b445175bff61b/packages/platforms/react-native/rollup.config.mjs#L21-L26